qemu/tests/device_option_check.py

```diff
@@ -100,10 +99,11 @@ def run(test, params, env):
     for check_cmd in check_cmds.split():
         check_cmd_params = params.object_params(check_cmd)
         cmd = check_cmd_params['cmd']
+        cmd = utils_misc.set_winutils_letter(session, cmd)
         pattern = check_cmd_params['pattern'] % parameter_value_raw
 
-        error.context("Check option with command %s" % cmd, logging.info)
-        _, output = session.cmd_status_output(cmd)
+        error_context.context("Check option with command %s" % cmd, logging.info)
+        status, output = session.cmd_status_output(cmd)
```

Comment:
`_, output = session.cmd_status_output(cmd)` this indicates it will not need the `status`. That's why it was code like that. You don't need to change this. or you can use it this way `output = session.cmd_status_output(cmd)[1]`